src/fullscreen/fullscreen-controller.js

```diff
@@ -19,6 +19,8 @@ class FullscreenController {
   _player: Player;
   _isInBrowserFullscreen: boolean;
   _eventManager: EventManager;
+  // Flag to overcome browsers which supports more than one fullscreenchange event
+  _isFullscreenEventDispatched: boolean = false;
```

Comment:
Which browsers does it?
It was in the past and I removed it since shaka polyfill cause this issue.

Reply:
Has nothing to do with Shaka AFAIK. 
Happens on Safari which supports `fullcreenchange` and `webkitfullscreenchange`
and on IE/Edge  which supports `fullcreenchange` and `msfullscreenchange`.

Reply:
Shaka fixed this polyfill on their side

Reply:
these are double events we getting from registering them all.
We will open a task to register the correct events like screenfull library is doing.